hadoop-common-project/hadoop-kms/src/test/java/org/apache/hadoop/crypto/key/kms/server/TestKMS.java

```diff
@@ -929,6 +930,7 @@ public Void call() throws Exception {
   }
 
   @Test
+  @SuppressWarnings("unchecked")
```

Comment:
I add @SuppressWarnings("unchecked") in this position, it will not bring additional type conversion risk.